docs/tutorials/Build_Two_Parties_FATE_Cluster_in_One_Linux_Machine_with_MiniKube.md

```diff
@@ -255,7 +255,11 @@ modules:
   - fateboard
   - client
```

Comment:
Let's revert this file, I have changed this in another pr.

Reply:
done